voidcrew/code/modules/factions/_common/faction_hud.dm

```diff
@@ -0,0 +1,68 @@
+/datum/atom_hud/faction
+	hud_icons = list(FACTION_HUD) //the icon list for factions
+	var/self_visible = TRUE //if the player can view there own hud
+	var/icon_color //will set the icon color to this
```

Comment:
```suggestion
    ///will set the icon color to this
	var/icon_color
```
any new variable likes this should have a comment on the line before it with `///`